connectors/sources/sharepoint_online.py

```diff
@@ -66,9 +66,10 @@
     GRAPH_API_AUTH_URL = "https://login.microsoftonline.com"
     REST_API_AUTH_URL = "https://accounts.accesscontrol.windows.net"
 
-DEFAULT_RETRY_COUNT = 3
+DEFAULT_RETRY_COUNT = 5
```

Comment:
Should we make these configurable as part of this? Even as advanced configs?

Reply:
Yes, I think we should. I talked about this with @danajuratoni so I'll let her decide. If we make it configurable I'd like to do it in a separate PR.